### DISCUSSION TITLE
本科论文结构（绪论/引言、结束语、结论）问题

### DISCUSSION THREAD

**Discussion by ghost** [note286/xduts#221] [Q&A] [answered]
学长您好，学校下发的本科毕业论文撰写规范中要求：“毕业设计（论文）的主体部分包括引言（绪论）、正文、结论、结束语、致谢、参考文献。”，![屏幕截图 2024-12-31 095048](https://github.com/user-attachments/assets/831bfa33-a32d-48a4-ae1a-034a91d4bf26)
![屏幕截图 2024-12-31 095039](https://github.com/user-attachments/assets/ef4b4258-fbd1-488f-97f5-6da2d6705fb2)
xduts模板中单列致谢、参考文献，那么是否引言、结论和结束语都需要单列（即不在\begin{document}后使用\chapter{引言}这样列出，而是像致谢和参考文献一样不存在一级标题），
我应该如何修改.cls文件呢？
pdf预览：
![屏幕截图 2024-12-31 095026](https://github.com/user-attachments/assets/543c9dbe-ba49-44fa-a4f7-d0b9cd83a6db)

**Comment by note286** [comment, marked as answer]
引言、结论、结束语这三个，个人认为纯纯是教务处瞎搞，正常论文没有人写结束语这一章的，前两个也差不多，你无视即可，不用过于参考教务处的要求。如果还有疑虑可以问问你的毕设指导教师。

致谢和参考文献是必须的，而且是无需章节序号的，其余的都属于正文，因此需要章节序号，而且每个人的正文都是不同的，所以直接使用`\chapter{}`即可。

**Reply by ghost** [reply]
好的，感谢学长~（我个人也认为教务处工作效率低工作质量低表面工作要求高）
